umm-spec-lib/src/cmr/umm_spec/url.clj

```diff
@@ -5,10 +5,9 @@
   (:import
    (java.net URL MalformedURLException)))
 
-#_{:clj-kondo/ignore [:non-arg-vec-return-type-hint]}
-(defn ^URL url
+(defn url
   "Attempts to return a URL from the provided value. Returns nil if the value is not a URL."
-  [x]
+  [^URL x]
```

Comment:
This doesn't seem equivalent. This type hint is being applied to the input arg rather than the function, isn't it?

Reply:
Oops, you're correct. My mistake; correcting